scripts/utils/createChangesetQuestions.ts

```diff
@@ -0,0 +1,41 @@
+const config = {
+  type: "table",
+  name: "packages",
+  message: "Choose changed package and  a version to upgrade",
```

Comment:
There is extra whitespace between 'package and' and 'a version', and I think package should be plural
```suggestion
  message: "Choose the changed packages and a version to upgrade",
```